README.md

```diff
@@ -33,8 +33,7 @@ There is no web API to easily render complex layouts of text and other content i
 
 `drawHTMLElement(element ...)` takes the CTM (current transform matrix) of the canvas into consideration. The image drawn into the canvas is sized to `element`'s [`devicePixelContentBox`](https://web.dev/articles/device-pixel-content-box); element outsize that bounds (including ink and layout overflow) are clipped. The `drawHTMLElement(element, x, y, dwidth, dheight)` variant resizes the image of `element`'s subtree to `dwidth` and `dheight`.
 
-In an addition, a `fireOnEveryPaint` option is added to `ResizeObserverOptions`, allowing script to be notified whenever the drawn elements might have changed their
-DOM state and the canvas should be redrawn. The callback to the resize obsever will be called at resize obsever timing, which is after DOM style and layout, but before paint.
+In an addition, a `fireOnEveryPaint` option is added to `ResizeObserverOptions`, allowing script to be notified whenever elements under a `<canvas>` may render differently, so they can be redrawn. The callback to the resize obsever will be called at resize obsever timing, which is after DOM style and layout, but before paint.
```

Comment:
small typo: "obsever" -> "observer", twice.
Also, would this option only apply for children of canvas or is that sentence just saying that it will be useful for that use case among others?

Reply:
lgtm, thanks. And I agree the name can be bikeshed later.